### DISCUSSION TITLE
Ch #2

### DISCUSSION THREAD

**Discussion by smriti-mittal-au3** [browserengineering/book#997] [Q&A]
I have implemented the chapter, but a scrollbar doesn't show. Do we need to do something on top of the current code / implementation to be able to see a scrollbar ?

**Comment by Paganiniana** [comment]
Hey, @smriti-mittal-au3! I see that a scrollbar is listed in the [exercises](https://browser.engineering/graphics.html#exercises) for Chapter 2. That means that it shouldn't happen automatically, but requires something in addition to the chapter's code to make happen. Have you already implemented it and you're having some trouble getting it to show up?
